src/common/services/api/order.service.spec.js

```diff
@@ -733,6 +733,7 @@ describe('order service', () => {
       self.orderService.submit()
         .subscribe((data) => {
           expect(data).toEqual(purchaseResponse)
+          done()
```

Comment:
It's not enough to add `done()` here, you also have to send it as a parameter in the `it`